README.md

```diff
@@ -2,28 +2,35 @@
 
 ![Screenshot](./img/qgis-planet.webp)
 
-By Tim Sutton and QGIS Contributors.
+This project aggregates various RSS feeds related to the project, providing a centralized hub for updates and news.
+
+It has been mainly developed by Tim Sutton, Lova Andriarimalala and contributors.
 
 ## Project Status
 
 [![Deploy Hugo site to Pages](https://github.com/qgis/QGIS-Planet-Website/actions/workflows/github-pages.yml/badge.svg)](https://github.com/qgis/QGIS-Planet-Website/actions/workflows/github-pages.yml)
 ![Website Status](https://img.shields.io/website-up-down-green-red/http/qgis.github.io/QGIS-Planet-Website.svg)
 ![License](https://img.shields.io/github/license/qgis/QGIS-Planet-Website.svg)
 
-![](https://img.shields.io/github/issues/qgis/QGIS-Planet-Website.svg)
-![](https://img.shields.io/github/issues-closed/qgis/QGIS-Planet-Website.svg)
-![](https://img.shields.io/github/issues-pr/qgis/QGIS-Planet-Website.svg)
-![](https://img.shields.io/github/issues-pr-closed/qgis/QGIS-Planet-Website.svg)
-
-
 Project stars:
+
 ![Stars](https://starchart.cc/qgis/QGIS-Planet-Website.svg)
 
-## Subscriber Model
+## How to be referenced
+
+To make your website appear on this Planet:
+
+- [ ] a website publishing a public and compliant RSS feed (we use [feedparser](https://feedparser.readthedocs.io/) under the hood) with content about the QGIS project
+- [ ] add your RSS feed in the subscribers list using a Pull Request (or an issue if you don't know how to do a fork + PR). See [below for the subscriber model](#subscriber-model).
+
+> [!NOTE]
+> Only feed items with, at least, the `QGIS` [category](https://www.rssboard.org/rss-specification#ltcategorygtSubelementOfLtitemgt) (= tag) will be listed. Even if a mechanism to customize tags per feed, it's disabled for now. [See this discussion](https://github.com/qgis/QGIS-Planet-Website/pull/49#discussion_r1958152569).
```

Comment:
"Even if a mechanism to customize tags per feed, it's disabled for now."

to

"Although there was work done on a mechanism to customize tags per feed, we have disabled this for now."